setup/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "notation-setup",
+  "version": "1.0.0",
+  "description": "Install and configure the Notation CLI and plugins.",
+  "main": "index.js",
+  "scripts": {
+  "scripts": {
+      "build": "ncc build index.js -o dist --source-map --minify --license licenses.txt",
```

Comment:
@duffney I see you updated the package.json, could you run the `npm run build` under `setup` dir?

Reply:
Ran `npm run build` and pushed up changes. :) Thank you for the instructions @Two-Hearts.